_includes/manuals/1.12/5.8.3_secret_token.md

```diff
@@ -0,0 +1,8 @@
+
+The last file required to make a Foreman cluster work is __$app_root/tmp/secret_token.rb__. This file should contain a secret token that is modeled after __$app_root/config/initializers/local\_secret\_token.rb__. __$app_root/tmp/secret_token.rb__ is used by __$app_root/config/initializers/secret_token.rb__ to sign cookies. Once you have created the new secret_token.rb, restart Foreman and clear Foreman's cache:
```

Comment:
Hm, that's peculiar. I'd suggest a couple of things:

1. If you'd like to investigate the issue, adding debugging to the top of secret_token.rb might help: `File.open(Rails.root.join('tmp', 'token.log'), 'a') { |f| f.puts "secret_token=#{Foreman::Application.config.secret_token}" }`, then check tmp/token.log matches the token set in local_secret_token.rb. It should be set if the file exists. I'd like to avoid recommending using tmp/secret_token if possible.
1. Failing that, this should recommend that local_secret_token.rb is set identically across all servers, and that `tmp/secret_token` is also set identically if it exists.
1. Also, `tmp/secret_token.rb` should have the .rb extension removed, it's just `tmp/secret_token`